src/Routes/SeasonsController.ts

```diff
@@ -2,21 +2,101 @@ import * as express from "express";
 import { IDataProvider } from "../DataProviders/IDataProvider";
 import { BaseController } from "./BaseController";
 
-/*
- * Seasons controller
+/**
+ * @swagger
+ *
+ * definitions:
+ *   Seasons:
+ *     type: object
+ *     properties:
+ *       isDataSpecific:
+ *         type: boolean
+ *       name:
+ *         type: string
+ *       order:
+ *         type: integer
+ *       reasonId:
+ *         type: integer
+ *       verse:
+ *         type: string
+ *
+ * /seasons/:
+ *   get:
+ *     description: Gets a list of Seasons
+ *     produces:
+ *       - application/json
+ *     responses:
+ *       200:
+ *         description: Seasons
+ *         schema:
+ *           type: array
+ *           items:
+ *             $ref: '#/definitions/Seasons'
+ *
  */
 export class SeasonsController extends BaseController {
     constructor(dataProvider: IDataProvider) {
         super();
-        this.router.get("/", async (req: express.Request, res: express.Response) => {
+        this.router.get("/seasons", async (req: express.Request, res: express.Response) => {
```

Comment:
leave this as /seasons.  The app adds all the controller's routes under /seasons.  see app.ts

Reply:
Done